scripts/install_deps.sh

```diff
@@ -299,6 +301,15 @@ case $(uname -s) in
                         echo "Installing solidity dependencies on Ubuntu Artful (17.10)."
                         install_z3="libz3-dev"
                         ;;
+                    betsy)
+                        #do not try anything for betsy.
+                        echo "Linux Mint Betsy is not supported at the moment as it runs off of Debian."
```

Comment:
A part of this script also deals with debian, so I think it should also work for betsy.